zerver/webhooks/travis/doc.md

```diff
@@ -0,0 +1,22 @@
+See your Travis CI build notifications in Zulip!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Finally, set up a webhook in your `.travis.yml` file:
+
+```
+notifications:
+webhooks:
+- {{ external_api_uri_subdomain }}/v1/external/travis?stream=travis&amp;topic=build-status&amp;api_key=abcdefgh
```

Comment:
The `&`s shouldn't be escaped as `&amp;` anymore, since Markdown supports them without problems.

Leaving them escaped here will cause problems when setting up the integration (the querystring will have a bad format).